subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/git/GitPlugin.java

```diff
@@ -55,13 +57,15 @@
 
     public void apply(final Project project) {
         final ShipkitConfiguration conf = project.getPlugins().apply(ShipkitConfigurationPlugin.class).getConfiguration();
+        String commitMessage = generateCommitMessagePostfix(conf, getProperty("TRAVIS_BUILD_NUMBER"));
```

Comment:
GitPlugin should be decoupled from Travis, so that we can use it for Circle CI, etc. Take a look at #514 - we're adding new code to TravisPlugin.

Reply:
That's why I exposed it to CommitMessageUtil but using Travis Plugin for that seems to be really better way